symmetrix/utilities/create_symmetrix_mace.py

```diff
@@ -34,7 +42,7 @@ def linear_simplify(linear):
 
 ### ----- BASIC MODEL INFO -----
 
-num_elements = len(sys.argv) - 2
+num_elements = len(args.atomic_numbers)
```

Comment:
This will now be broken if `len(args.atomic_numbers) == 0` and it uses all those from the torch model.  I'll push a fix.

Reply:
In fact, is it worth storing `num_elements` in `output`, given that it contains the same info (via the length) as `output['atomic_numbers']`?